### DISCUSSION TITLE
Welcome to watching-testrunner Discussions!

### DISCUSSION THREAD

**Discussion by tony** [tony/watching-testrunner#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dwt** [comment]
I'm one of the original builders of watching test runner and use it every day still to automate my unit testing, but also to automate building arbitrary stuff like compiling sass files, building static sites on file changes... basically whenever the original tool doesn't have a nice / working watch functionality this nicely supplants it.
